frontend/src/App.tsx

```diff
@@ -876,7 +875,7 @@ export class App extends PureComponent<Props, State> {
       },
       () => {
         this.pendingElementsBuffer = this.state.elements
-        this.widgetMgr.removeInactive(fromJS([]))
+        this.widgetMgr.removeInactive(new Set([]))
```

Comment:
What is the reason why `fromJS` had to be changed here?

Reply:
In `baseui@9.116.3` `widgetMgr.removeInactive` recieves `Set` instead of `Immutable Collection`, the tschecks and compilation failed with previous code